.github/actions/push-registry-app-files-supabase/action.yml

```diff
@@ -0,0 +1,12 @@
+name: "Push registry app files to Supabase"
+description: "Push registry app files that have changed to Supabase. If any *app.ts files have changed, run pnpm build for the affected app to generate the *.app.mjs file."
+inputs:
+  changed_files:
+    description: "List of all changed files coming from the push_to_supabase job of a github action workflow"
+    required: true
+  supabase_anon_key:
+    description: "Supabase anon key"
+    required: true
+runs:
+  using: "node16"
```

Comment:
Small, but any reason we're on 16 here? We use 22 for most new projects / actions now.